packages/manager/modules/pci/src/projects/project/instances/add/add.controller.js

```diff
@@ -235,7 +235,8 @@ export default class PciInstancesAddController {
     this.isAddingPrivateNetwork = false;
     this.isAddingPrivateNetworkError = false;
 
-    this.fetch3AZAvailability();
+    this.regionAvailability = {};
```

Comment:
As you want to store multiple regions in a same object, I'd make it plural: 
```suggestion
    this.regionsAvailability = {};
```

Reply:
It is a region type not a region, but I can change it to regionTypesAvailability